src/tests/ftest/util/server_utils.py

```diff
@@ -382,6 +383,13 @@ def clean_files(self, verbose=True):
             if cmd not in clean_cmds:
                 clean_cmds.append(cmd)
 
+            # Remove the shared memory segment associated with each io server
+            for srv_idx in range(NUM_IO_SERVERS):
+                key = 0x10242048 + srv_idx
```

Comment:
This appears to be a redundant loop (see previous comment) and ultimately it will just set key = D_TM_BASE_KEY + len(NUM_IO_SERVERS) - 1.  Is that the intention?:
```suggestion
```

To get the equivalent of `srv_idx`, let's make this change on line 377:
```
for index, server_params in enumerate(self.manager.job.yaml.server_params):
```